src/AccessControl/tainted.py

```diff
@@ -169,3 +187,18 @@ def createOneOptArgWrapper(func):
 
 for f in oneOptArgWrappedMethods:
     setattr(TaintedString, f, createOneOptArgWrapper(f))
+
+
+class TaintedBytes(TaintedString):
+    
+    def __init__(self, value):
+        if isinstance(value, int):
+            value = bytes([value])
```

Comment:
Are you sure this works? It gives wildly different results in Python 2 and 3.

Python 3 looks okay:
```
>>> bytes([60, 61, 62])
b'<=>'
```
Python 2 is probably not what we want:
```
>>> bytes([60, 61, 62])
'[60, 61, 62]'
```

Reply:
Good point. Passing a int should happen only with Python 3.
I'll add a check.

Reply:
Won't this fail with an AssertionError exception? And IIRC assert statements might be removed during Python bytecode compiling or with extra optimization flags.

Reply:
My feeling is that an assert is enough.